frontend/src/components/Item/SeasonTabs.vue

```diff
@@ -5,34 +5,42 @@
         {{ season.Name }}
       </v-tab>
     </v-tabs>
-    <v-tabs v-model="currentTab" class="bg-transparent">
-      <v-tab v-for="season in seasons" :key="season.Id">
+    <v-window v-model="currentTab" class="bg-transparent">
+      <v-window-item v-for="season in seasons" :key="season.Id">
         <v-list
           v-if="seasonEpisodes && season.Id"
-          lines="two"
-          color="transparent">
+          lines="false"
```

Comment:
```suggestion
          :lines="false"
```
This prop accepts the boolean false, not the string false ;)
Also, is this needed at all?

Reply:
Jeezz, I couldn't figure out why it kept complaining. thanks

It is needed because otherwise it puts the subtitle (episode overview in this case) on one line and shortens the rest with "..."

Reply:
Oh great then! And no problem, once you know that happens you'll see it more quickly next time :)